iot/generate_keys.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This file isn't referenced anywhere, as far as I can find

How do I use it? Could you add something to the README about this, eg. an example of running this and then calling one of the sample commands?

Reply:
Added to README